mesh/accountsettings/accountsettings.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS AccountSettings(
+	settingID INT PRIMARY KEY AUTO_INCREMENT,
+    accountID INT,
+    isVerified BOOL NOT NULL DEFAULT FALSE,
+    enableContentFilter BOOL NOT NULL DEFAULT FALSE,
+    displayTheme ENUM("light","dark") NOT NULL DEFAULT "light",
+    is2FAEnabled BOOL NOT NULL DEFAULT FALSE,
+    phoneNum VARCHAR(15),
```

Comment:
Have you checked with @peppacaiou about taking these away from the `Accounts` table? Their PR #143 already captures these in `Accounts`.

Reply:
Just checked, good to go!